.env.example

```diff
@@ -0,0 +1 @@
+# Place any custom/local configuration here
```

Comment:
This app doesn't actually use the `.env.example` -- and local config would go in the `.env`, is this file necessary?

Reply:
Or rather, it is only used by the setup script-- maybe the script can check for this file to exist rather than including an empty file by default.

Reply:
This is a convention I borrowed from other projects, where the `.env.example` serves as a template from which to initialize the `.env` which is `.gitignore`d. While it's true Horizon doesn't need it, I'm hoping this PR can be a reference for other projects that will need it.